plugin.json

```diff
@@ -2,7 +2,11 @@
   "id": "skype4business",
   "name": "Skype for Business",
   "description": "Skype for Business plugin for Mattermost 5.2+.",
+  "homepage_url": "https://github.com/mattermost/mattermost-plugin-skype4business",
+  "support_url": "https://github.com/mattermost/mattermost-plugin-skype4business/issues",
+  "icon_path": "assets/profile.svg",
   "version": "0.1.2",
+  "min_server_version": "5.10.0",
```

Comment:
What is the process for choosing the `min_server_version`?

Reply:
Good question. The "latest" API method I could found was `GetBundlePath`. @kosgrz Could you double check that I didn't missed anything?

Reply:
@hanzei I've checked all APIs methods and `GetBundlePath` is the newest one :+1: